dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -290,13 +290,33 @@ class ScriptsControllerTest < ActionController::TestCase
       script: {name: script.name},
       script_text: '',
       pilot_experiment: 'pilot-experiment',
-      hidden: false,
+      visible_to_teachers: true,
```

Comment:
This param controls whether the script is hidden. see: https://github.com/code-dot-org/code-dot-org/blob/6635ab1ba8af1aa06f0b58c176bde3b941360e7b/dashboard/app/controllers/scripts_controller.rb#L151